From 7e55ba8e997eefd59a960fefd8bd40cb704f4832 Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Sat, 7 Jul 2018 18:40:55 +0200
Subject: [PATCH 18/25] msm8953-common: Remove send_mic_mute_to_AudioManager
 overlay

* send_mic_mute_to_AudioManager hasn't been doing anything
  since Android 4.4, therefore there's no reason to keep it.

Change-Id: I74ed33265fccd59def5c2f9e67a9e8a28ce4dbc9
---
 overlay/packages/services/Telephony/res/values/config.xml | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/overlay/packages/services/Telephony/res/values/config.xml b/overlay/packages/services/Telephony/res/values/config.xml
index c873054..883da92 100644
--- a/overlay/packages/services/Telephony/res/values/config.xml
+++ b/overlay/packages/services/Telephony/res/values/config.xml
@@ -1,9 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright (C) 2009 The Android Open Source Project
+
      Licensed under the Apache License, Version 2.0 (the "License");
      you may not use this file except in compliance with the License.
      You may obtain a copy of the License at
+
           http://www.apache.org/licenses/LICENSE-2.0
+
      Unless required by applicable law or agreed to in writing, software
      distributed under the License is distributed on an "AS IS" BASIS,
      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
@@ -23,11 +26,6 @@
          successfully placing the OTA call -->
     <integer name="OtaShowListeningScreen">1</integer>
 
-    <!-- Determine whether calls to mute the microphone in PhoneUtils
-         are routed through the android.media.AudioManager class (true) or through
-         the com.android.internal.telephony.Phone interface (false). -->
-    <bool name="send_mic_mute_to_AudioManager">true</bool>
-
     <!-- If this is true, the SIM card (through Customer Service Profile EF file) will be
          able to prevent manual operator selection. If false, this SIM setting will be
          ignored and manual operator selection will always be available. See
-- 
2.7.4

